core/templates/components/skill-selector/skill-selector.component.ts

```diff
@@ -194,9 +195,14 @@ export class SkillSelectorComponent implements OnInit {
   }
 
   searchInUntriagedSkillSummaries(searchText: string): SkillSummary[] {
-    let skills: string[] = this.untriagedSkillSummaries.map(val => {
-      return val.description;
-    });
+    let skills: string[] = this.untriagedSkillSummaries
+      .filter(
+        val => !this.skillIdsToExclude ? true :
+        !this.skillIdsToExclude.hasOwnProperty(val.id)
```

Comment:
Something seems wrong here. skillIdsToExclude is typed above as a list of strings but you're accessing it here as a JS object ("hasOwnProperty").

Reply:
Done